CHANGES.txt

```diff
@@ -182,7 +182,17 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
       rewordings in manpage.  Manpage Examples moved to an external
       repository / website (scons-cookbook.readthedocs.io).
 
-
+  From Joseph Brill:
```

Comment:
These blurbs are very internal to SCons focused. Ideally CHANGES.txt would be user focused.
So they can read the entries and say. oh the problem I have has been fixed in this version.

Can you rewrite the below with that in mind?